File: web/player.js
/**
 * Player
 */

/**
 * Global Variables
 * @type type
 */
//var currentPlayer;


/**
 * Return a new player object.
 * @param {type} id
 * @param {type} name
 * @param {type} status
 * @param {type} score
 * @returns {createPlayer.newplayer}
 */
function createPlayer(id, name, status, score) {
	
    var newplayer = {
	id: id,
	name: name,
	choice: "",
	status: status,
        score: score,
        setId: function(id) {
            this.id = id;
        },
        getId: function() {
            return this.id;
        },
        setName: function(name) {
            this.name = name;
        },
        getName: function() {
            return this.name;
        },
        setChoice: function(choice) {
            this.choice = choice;
        },
        getChoice: function() {
            return this.choice;
        },
        setStatus: function(status) {
            this.status = status;
        },
        getStatus: function() {
            return this.status;
        },
        setScore: function(score) {
            this.score = score;
        },
        getScore: function() {
            return this.score;
        }
    }
    return newplayer;
}

File: src/java/com/rockscissorspaper/model/Player.java
package com.rockscissorspaper.model;

public class Player {
	
	private int id;
	private String name;
	private int score;
	private int status;
	private String password;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public int getScore() {
		return score;
	}
	public void setScore(int score) {
		this.score = score;
	}
	public int getStatus() {
		return status;
	}
	public void setStatus(int status) {
		this.status = status;
	}
	public void setPassword(String password) {
            this.password = password;
        }
	public String getPassword() {
            return this.password;
        }
	
}
